python/smashbox/utilities/ocutil.py

```diff
@@ -0,0 +1,79 @@
+from smashbox.script import config
+from smashbox.utilities import *
+
+def configure_storage ():
+    """ Update the database to support users creating their own external storage 
+        mount points.  This is in place of an API and should be changed once the
+        api and/or occ support is available"""
+
+    db = None;
+    db_schema = config.get('db_schema', 'owncloud')
+    db_host = config.get ('db_host', '172.18.5.57')
+    db_user = config.get ('db_user', 'root')
+    db_password = config.get('db_password', 'Password01')
+
+    try:
+
+        db = MySQLdb.connect (db_host, db_user, db_password)
```

Comment:
what if we are not using mysql?!
In 8.2 we will have an occ tool for that, which can be used with the server shell command.